src/migratesettings.h

```diff
@@ -19,4 +19,12 @@ void migrate_settings()
 		g_settings->setBool("touch_gui", value);
 		g_settings->remove("enable_touch");
 	}
+
+	// Disables anticheat
+	if (g_settings->existsLocal("disable_anticheat")) {
+		if (g_settings->getBool("disable_anticheat")) {
+			g_settings->set("anticheat_flags", "nodigging,nointeraction,nomovement");
+		}
```

Comment:
I'd prefer

```suggestion
		if (g_settings->getBool("disable_anticheat"))
			g_settings->setFlagStr("anticheat_flags", 0, flagdesc_anticheat);
```

otherwise, you're "working around the abstraction"

Reply:
Will have to add `#include "server.h"` to migratesettings.h too though...